app/assets/stylesheets/content/_forms.sass

```diff
@@ -328,6 +331,9 @@ fieldset.form--fieldset
   @extend %form--field-after-container
   font-style: italic
 
+  &.-no-italic
+    font-style: normal
```

Comment:
One issue I have with this modifier is that its name carries no semantic information (as in: why should I use no italics), but only the mere styling information: "use no italics, don't ask why"...

Still I have no better idea :/